contentcuration/contentcuration/frontend/shared/views/Sandbox.vue

```diff
@@ -3,17 +3,36 @@
   <VApp>
     Insert component here for testing!
     <TemplateComponent />
+
```

Comment:
Remove the changes here, so they don't mess with other people's sandbox

Reply:
ok I dropped that last commit and force pushed